scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,100 @@
+# IPC gullwing generator
+
+This generator uses IPC-7351B equations and by default fillet definitions to derive a footprint for gullwing stile packages.
+Examples of supported packages include: QFN, SOIC, SO
+
+## Running the script
+
+The script requires python 3.4 or newer. Run it with:
+`python3 ipc_gullwing_generator.py size_definitions/eqfp.yaml` (replace eqfp.yaml with the size definition file that contains your part.)
+
+## Size definition format
+
+Every file contains a header with parameters applied to all parts. These define the common footprint name prefix and the output library.
+
+```yaml
+FileHeader:
+  library_Suffix: 'QFP' #resulting library name Package_QFP
+  device_type: 'EQFP' #footprint names will be of style ...EQFP-pincount_...
+```
+
+---
+
+Every further entry in the script is assumed to be a package size definition. The top level parameter will give the internal parameter set name. This one must be unique in this file and should be representative of the footprint as it will be used in error messages. (It also makes later maintenance easier if it is easy to determine which footprint was genererated by which parameter set.)
+
+``` yaml
+internal_name:
+  # parameter list
+```
+
+The following parameters are supported for every package
+- Size source to be added to footprint documentation field (`size_source`) {url}
+- footprint name control
+  - Manufacturer if specific to one manufacturer (`manufacturer`) {string}
+  - Part number if specific to one part (`part_number`) {string}
+  - Custom naming (`custom_name_format`) {format string}
+- Body size (`body_size_x`, `body_size_y`, `overall_height`) {dimension}
+- Lead dimensions:
+  - Over all size representing lead tip to lead tip dimension (`overall_size_x`, `overall_size_y`) {dimension}
+  - Lead with (`lead_width`) {dimension}
+  - Lead length representing tip till bend (`lead_len`)
+- lead pitch currently equal for all sides (`pitch`) {float}
+- pin count, 0 means no pins in that direction (`num_pins_x`, `num_pins_y`) {int}
+- Exposed pad:
+  - Size of package exposed pad or slug (`EP_size_x`, `EP_size_y`) {dimension}
+  - Optional the size of the footprint pad (`EP_size_x_overwrite`, `EP_size_y_overwrite`) {float}
+  - Optional the size of the mask cutout (`EP_mask_x`, `EP_mask_y`) {float}
+  - Paste can be split into regular grid with (`EP_num_paste_pads`) {[int (x), int (y)]}
+    - Paste coverage multiplicator (`EP_paste_coverage`) {float (0..1)}
```

Comment:
Should 'multiplicator' be 'percentage'?

Reply:
percentage would imply a value between 0 and 100. This value is the result of dividing two values (The overall area divided by the paste area) without multiplying senselessly with 100.

Reply:
Then the word should be 'multiplier'.

Reply:
fixed